subjects/argsort/README.md

```diff
@@ -0,0 +1,29 @@
+## argsort
+
+### Instructions
+
+Write a program that checks if an argument is sorted or not.
+- Your Program should return `T` followed by a newline (`'`\n'`) if an argument is sorted.
+- Your Program should return `F` followed by a newline (`'`\n'`) if an argument is not sorted.
+- Space and Tabulation are included in the sort.
```

Comment:
What does this means? We could interpret this in different ways so explain it a bit more detailed